presto-main/src/main/java/io/prestosql/server/ServerMainModule.java

```diff
@@ -191,6 +182,7 @@ protected void setup(Binder binder)
         configBinder(binder).bindConfig(FeaturesConfig.class);
 
         binder.bind(SqlParser.class).in(Scopes.SINGLETON);
+        SqlParserOptions sqlParserOptions = new SqlParserOptions();
```

Comment:
also please move `sqlParserOptions.useEnhancedErrorHandl...` to before guice bind